framework/test/driver/ServiceTrackerTest.cpp

```diff
@@ -33,33 +33,25 @@
 #include "TestingConfig.h"
 #include "TestingMacros.h"
 
+#include <chrono>
 #include <future>
 #include <memory>
-#include <chrono>
 
 using namespace cppmicroservices;
 
 bool CheckConvertibility(const std::vector<ServiceReferenceU>& refs,
                          std::vector<std::string>::const_iterator idBegin,
```

Comment:
Why take iterators when the copy is going to happen anyway. Why not const std::vector& as a parameter and construct the vector at the call site rather than inside this method